qemu/tests/cfg/timerdevice.cfg

```diff
@@ -58,6 +58,10 @@
                 - base_localtime:
                     only Windows
                     rtc_base = localtime
+                - base_date:
+                    Windows:
+                        only Win2000, WinXP, Win2003, WinVista, Win7, Win2008
```

Comment:
Could you pls explain why specify these windows guest here?  I think the boot and reboot cases are applicative for win2012r2, win2019, etc. So no need to only here.